py/sync_examples.py

```diff
@@ -112,9 +112,10 @@ def main():
 
     example_items = [dict(slug=e.slug) for e in examples]
     example_items.insert(0, dict(slug='index'))
+    example_items.insert(1, dict(slug='gallery'))
     write_file(os.path.join(website_dir, 'examples.js'), f'module.exports={json.dumps(example_items)}')
-    # write_file(os.path.join(example_md_dir, 'index.md'), make_toc(examples))
-    write_file(os.path.join(example_md_dir, 'index.md'), make_gallery(examples))
+    write_file(os.path.join(example_md_dir, 'index.md'), make_toc(examples))
+    write_file(os.path.join(example_md_dir, 'gallery.md'), make_gallery(examples))
```

Comment:
It would be good to preserve the gallery page as-is, and add a new page for the ToC.

Reply:
Done